docs/source/release/v6.5.0/index.rst

```diff
@@ -16,7 +16,25 @@ Mantid 6.5.0 Release Notes
 
 We are proud to announce version 6.5.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+This release includes a wide array of updates, bugfixes, and new features, which build on the functionality of Mantid.
+Users are reminded that since the previous release the :ref:`packaging method for Linux has changed <v6.4.0>`.
+
+In addition to many improvements we are delighted to announce some new features including:
+
+- A new tab has been added to the :ref:`ISIS Engineering Diffraction UI<Engineering_Diffraction-ref>` to support running refinements in GSAS-II.
+- The ISIS powder diffraction scripts support the PaalmanPings correction method.
+- It is now possible to use the algorithms from the :ref:`GOFit optimization package <gofit-fitting>` directly with the :ref:`Crystal Field API <Crystal Field Python Interface>`.
+- The algorithm :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` now simulates scattering in a container (and more generally in any sample environment components) if a user sets up the shapes\materials in the input workspace prior to running the algorithm.
```

Comment:
Sphinx has removed the `\` because it assumes `\m` to be an escape character. Maybe replace this part with:

*if a user sets up the shapes and materials*

?